docs/_interface/JavaScript-Console.md

```diff
@@ -1,83 +1,173 @@
 
-(note: the web3 version bundled within geth is very old, and not up to date with official docs)
+```
+curl --data '{"jsonrpc":"2.0","method":"eth_getBalance", "params": ["0x9b1d35635cc34752ca54713bb99d38614f63c955", "latest"], "id":2}' -H "Content-Type: application/json" localhost:8545
 
-## Interactive Use: The Console
+```
 
-The geth JavaScript console is started with the `console` or `attach` geth sub-commands.
-The `console` subcommands starts the geth node and then opens the console. The `attach`
-subcommand attaches to the console to an already-running geth instance.
+This returns a result which is also a JSON object, with values expressed as hexadecimal strings, for example:
 
-    geth console
-    geth attach
+```terminal
 
-Attach mode accepts an endpoint in case the geth node is running with a non default
-ipc endpoint or you would like to connect over the rpc interface.
+{"id":2,"jsonrpc":"2.0","result":"0x1639e49bba16280000"}
 
-    geth attach /some/custom/path.ipc
-    geth attach http://191.168.1.1:8545
-    geth attach ws://191.168.1.1:8546
+```
 
-Note that by default the geth node doesn't start the HTTP and WebSocket servers and not
-all functionality is provided over these interfaces for security reasons. These defaults
-can be overridden with the `--http.api` and `--ws.api` arguments when the geth node is
-started, or with [admin.startRPC](../rpc/ns-admin#admin_startrpc) and
-[admin.startWS](../rpc/ns-admin#admin_startws).
+While this approach is valid, it is also a very low level and rather error-prone way to interact with Geth. Most developers prefer to use convenience libraries that abstract away some of the more tedious and awkward tasks such as converting values from hexadecimal strings into numbers, or converting between denominations of ether (Wei, Gwei, etc). One such library is [Web3.js](https://web3js.readthedocs.io/en/v1.7.3/). This is a collection of Javascript libraries for interacting with an Ethereum node at a higher level than sending raw JSON objects to the node. The purpose of Geth's Javascript console is to provide a built-in environment to use a subset of the Web3.js libraries to interact with a Geth node.
 
-If you need log information, start with:
+{% include note.html content="The web3.js version that comes bundled with Geth is not up to date with the official Web3.js documentation. There are several Web3.js libraries that are not available in the Geth Javascript Console. There are also administrative APIs included in the Geth console that are not documented in the Web3.jc documentation. The full list of libraries available in the Geth console is available on the [JSON-RPC API page](/docs/_rpc/server)" %}
```

Comment:
Is there no alternative way to includes notes like these? I would prefer not to mix markup with text, as much as possible. 

I think I'd prefer just having the note as plain text rather than in such a tag. But that's just my 5c

Reply:
afaik the only pure markdown options available are backticks, bold/italic text or quote blocks. Only the quote block is really viable in my opinion but none are really sufficiently eye-catching.

> example quote block